src/client/public/shared/helpers/password-toggle.ts

```diff
@@ -0,0 +1,20 @@
+export function initPasswordToggle() {
+	const passwordInput = document.getElementById("password");
+	const togglePasswordBtn = document.getElementById("togglePassword");
+	const showPasswordIcon = togglePasswordBtn.querySelector(
+		".show-password-icon",
+	);
+
+	togglePasswordBtn.addEventListener("click", () => {
+		const type =
+			passwordInput.getAttribute("type") === "password" ? "text" : "password";
+		passwordInput.setAttribute("type", type);
+
+		// Change the icon based on the password visibility
+		if (type === "password") {
+			showPasswordIcon.setAttribute("src", "/icons/eye-closed.svg");
+		} else {
+			showPasswordIcon.setAttribute("src", "/icons/eye-visible.svg");
+		}
+	});
+}
```

Comment:
use dom module that we have in `/client/src/shared/libs/modules/dom/dom.ts`, add a new method to it for changing attributes of DOM elements